src/Microsoft.DotNet.Build.Tasks/Microsoft.DotNet.Build.Tasks.csproj

```diff
@@ -4,35 +4,28 @@
   <PropertyGroup>
```

Comment:
@weshaggard:

> I thought the plan was to keep this project and make a new ".NETCore" csproj file so we could build it for either target.

The tasks compiled for .NET Core should work for both versions of MSBuild, so it shouldn't be necessary to compile the code twice.  Let us know if you run into any issues with this.